python/cugraph/components/connectivity.py

```diff
@@ -48,3 +49,39 @@ def weakly_connected_components(G):
     df = connectivity_wrapper.weakly_connected_components(G.graph_ptr)
 
     return df
+
+
+def strongly_connected_components(G):
+    """
+    Generate the stronlgly connected components and attach a component label to
+    each vertex.
+
+    Parameters
+    ----------
+    G : cugraph.Graph
+      cuGraph graph descriptor, should contain the connectivity information as
+      an edge list (edge weights are not used for this algorithm). The graph
+      can be either directed or undirected where an undirected edge is
+      represented by a directed edge in both directions.
+      The adjacency list will be computed if not already present.
+      The number of vertices should fit into a 32b int.
+
+    Returns
+    -------
+    df : cudf.DataFrame
+      df['labels'][i] gives the label id of the i'th vertex
```

Comment:
Need an empty line after this line for this to be properly formatted. If not this line and the next line appears in the same line in the generated html file.

Reply:
I'm sorry, if it passes flake8 tests, it's compliant. I don't see why you feel the need to enforce more rules than flake8.

Reply:
This is not about python code formatting. This is about automatic document generation.

After running python setup.py install, if you go to cugraph/doc and run make html. This will automatically generate html documentation under build/html.

If you open the html file, this will appear as
```
df['labels'][i] gives the label id of the i'th vertex df['vertices'][i] gives the vertex id of the i'th vertex
```
instead of
```
df['labels'][i] gives the label id of the i'th vertex
df['vertices'][i] gives the vertex id of the i'th vertex
```
An empty line is necessary to get the latter (if you have used latex, this works similar to latex).